### PR TITLE
Fixes for admin app issues/enhancements

### PR DESCRIPTION
**Pull request by brittnylapierre**
- [x] #298 
- [x] #299
- [x] #302
- [x] Documentation

### CODE REVIEW

File: services/admin/src/lib/components/dmd/DmdSuccessView.svelte [hunk 1]
```diff
@@ -169,7 +169,11 @@ This component shows the view for a dmd task that had its metadata successfully
     {:else}
       <NotificationBar
         status="fail"
-        message={`Every item's metadata is invalid. To get an idea of what went wrong for each item, press their preview button. You can use this information to make changes to ${
+        message={`${
+          dmdTask?.items?.length
+            ? "Every item's metadata is invalid. To get an idea of what went wrong for each item, press their preview button. You can use this information to m"
```
Comment (SaschaAdler) [inline services/admin/src/lib/components/dmd/DmdSuccessView.svelte:174]: I think you cut off this error string from what it was.

**Review by SaschaAdler — CHANGES_REQUESTED** [review]
Looks fine except for my one comment.

File: services/admin/src/lib/components/dmd/DmdSuccessView.svelte (hunk 1 above)
Comment (brittnylapierre) [reply services/admin/src/lib/components/dmd/DmdSuccessView.svelte:174]: Ah I kind of added a bit of inline logic, here's the full line:

`
${ dmdTask?.items?.length ? "Every item's metadata is invalid. To get an idea of what went wrong for each item, press their preview button. You can use this information to m" : "No items found. M" }ake changes to ${ dmdTask?.fileName?.length ? dmdTask?.fileName : "the file" , then try again by uploading the corrected file.`}
`

So the message depends on the items list length. I can make this a variable and use the $: codeblock instead to make it more clear?

Comment (SaschaAdler) [reply services/admin/src/lib/components/dmd/DmdSuccessView.svelte]: Oh, I see what you did. I'm not sure this is ideal, and I wouldn't recommend doing things this way for anything we wanted to provide in multiple languages, but it's fine for now.

**Review by SaschaAdler — APPROVED** [review]

File: services/admin/src/lib/components/dmd/DmdSuccessView.svelte (hunk 1 above)
Comment (brittnylapierre) [reply services/admin/src/lib/components/dmd/DmdSuccessView.svelte]: Ohh okay! I've never done multi language support on a website I'm excited to get there, thanks Sascha!

**Comment by brittnylapierre** [comment]
Hey @RussellMcOrmond since it was quick i added the polling here too, let me know if it looks good to you! I'll merge and trigger a new build if so

**Review by RussellMcOrmond — APPROVED** [review]
Given these are new things and not something that could break something live, I'm happy to review after a deploy to production.  I assume you ran both `pnpm dev` against dev data and `pnpm prod` against production data.

**Comment by brittnylapierre** [comment]
Tested it with pnpm run prod! I'll merge and build for production now

**Pull request merged**